contrib/libs2n.spec

```diff
@@ -0,0 +1,84 @@
+Summary: s2n is a C99 implementation of the TLS/SSL protocols that is designed to be simple, small, fast, and with security as a priority.
+Name: libs2n
+Version: 20150703
+Release: 1%{?dist}%{?extra_release}
+License: Apache 2.0
+Group: Development/Libraries
+Vendor: Amazon.com, Inc.
+Prefix: %{_prefix}
+BuildRoot: %{_tmppath}/%{name}-%{version}-root
+URL: https://github.com/awslabs/s2n
+Source: libs2n-%{version}.tar.gz
+BuildRequires: glibc-devel
+Requires: glibc
+Requires: zlib
+
+%description
+%{summary}
+
+%package devel
+Summary: Libraries, includes, etc. to compile with the libs2n library.
+Group: Development/Libraries
+Requires: libs2n = %{version}
+
+%description devel
+Includes and documentations for the C++ libs2n library.
+
+%prep
+%setup
+
+%build
+make %{?_smp_mflags} libs
+
+%install
+[ "${RPM_BUILD_ROOT}" != "/" ] && rm -fr "${RPM_BUILD_ROOT}"
+install -D -m 0755 api/s2n.h ${RPM_BUILD_ROOT}%{_includedir}/s2n.h
+install -D -m 0755 lib/libs2n.so ${RPM_BUILD_ROOT}%{_libdir}/libs2n.so
+install -D -m 0755 lib/libs2n.a ${RPM_BUILD_ROOT}%{_libdir}/libs2n.a
+install -D -m 0755 contrib/libs2n.pc ${RPM_BUILD_ROOT}%{_libdir}/pkgconfig/libs2n.pc
```

Comment:
The header and pc files do not need to be executable. That's a mistake in the spec file.